docs/reference/portal.dataProduct.json

```diff
@@ -560,7 +560,7 @@
                         },
                         {
                             "name": "generalSettings",
-                            "label": "General Settings",
+                            "label": "Analytics Service Settings",
```

Comment:
[Portal Template Invalid](/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fdata-product-streaming%2Fbjcmit%2Fdata_observability%2Finfra%2Fmain.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fdata-product-streaming%2Fbjcmit%2Fdata_observability%2Fdocs%2Freference%2Fportal.dataProduct.json)

Reply:
What is your suggested change?

Reply:
Should the label be "General Settings"?